staging/podsecuritypolicy/rbac/README.md

```diff
@@ -32,11 +32,13 @@ It is important to note that this example uses the following syntax to test with
 
 1.  `--server=https://127.0.0.1:6443`: when performing requests this ensures that the protected port is used so
 that RBAC will be enforced
-1.  `--token={user}/{group(s)}`: this syntax allows a request to specify the username and groups to use for
-testing.  It relies on the `ALLOW_ANY_TOKEN` setting.
+1.  `--token=<token>`: this allows to make requests from a different users during testing.
```

Comment:
- As of `ALLOW_ANY_TOKEN` was completely removed, what is the supposed workflow now?
- How do I create unprivileged user to be able to use its token for the tests?
- Does `local-up-cluster.sh` support it out-of-box? (I tried to use `--token-auth-file` but 1) it didn't work 2) I had to manually editing `local-up-cluster.sh` because there is no way to pass this option via env variables.)

CC @ericchiang

Reply:
Impersonation headers are the recommended way of doing this

```
kubectl create -f example-pod.yaml --as jane
```

Reply:
```console
$ kubectl create -f examples/podsecuritypolicy/rbac/pod.yaml --as jane
Error from server (Forbidden): error when creating "examples/podsecuritypolicy/rbac/pod.yaml": pods is forbidden: User "jane" cannot create pods in the namespace "default"
```
How to bootstrap a token for the "jane"?

Reply:
The error is indicating that you haven't given the user "jane" any permissions. You still need to create an RBAC binding to that user.

```
kind: ClusterRoleBinding
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: read-secrets-global
subjects:
- kind: User
  name: jane
  apiGroup: rbac.authorization.k8s.io
roleRef:
  kind: ClusterRole
  name: edit
  apiGroup: rbac.authorization.k8s.io
```

Reply:
Ok, thanks! I'll use `kubectl create clusterrolebinding` command.

Reply:
Hm.. Couldn't make it work:
```console
$ kubectl create -f examples/podsecuritypolicy/rbac/pod.yaml --as jane
Error from server (Forbidden): error when creating "examples/podsecuritypolicy/rbac/pod.yaml": pods is forbidden: User "jane" cannot create pods in the namespace "default"
$ kubectl create clusterrolebinding restrict-jane --clusterrole=restricted-psp-user --user=jane
clusterrolebinding "restrict-jane" created
$ kubectl create -f examples/podsecuritypolicy/rbac/pod.yaml --as jane
Error from server (Forbidden): error when creating "examples/podsecuritypolicy/rbac/pod.yaml": pods is forbidden: User "jane" cannot create pods in the namespace "default"
```

Reply:
"jane" can use the policy but can't create pods?

```
kubectl create clusterrolebinding editors --clusterrole=edit --user=jane
```

Reply:
@ericchiang Yes; thanks!